pkg/loop/internal/core/services/capability/capabilities_registry.go

```diff
@@ -202,6 +237,50 @@ func (c *capabilitiesRegistryServer) Get(ctx context.Context, request *pb.GetReq
 	}, nil
 }
 
+func (c *capabilitiesRegistryServer) GetLocalNode(ctx context.Context, _ *emptypb.Empty) (*pb.GetLocalNodeReply, error) {
+	node, err := c.impl.GetLocalNode(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// convert workflowDON to pb.DON
+	membersBytes := make([][]byte, len(node.WorkflowDON.Members))
+	for i, m := range node.WorkflowDON.Members {
+		m := m
+		membersBytes[i] = m[:]
+	}
+	workflowDONpb := &pb.DON{
+		Id:      node.WorkflowDON.ID,
```

Comment:
If a node doesn't belong to a workflow DON, what do these values look like? I might have missed it, but is this case covered by unit testing?

Reply:
Good question, i dont know either lol

It should be zero values but i should check the (de)serialization logic via test 

https://github.com/smartcontractkit/chainlink-common/pull/620/files#diff-8c726a1dbd12cccdfdc7bd212b79d5bcc20cf100a7298f933e05e14dbbcd4288R196